libafl/src/bolts/os/unix_signals.rs

```diff
@@ -13,46 +13,76 @@ use nix::errno::{errno, Errno};
 #[cfg(feature = "std")]
 use std::ffi::CString;
 
-/// armv7 `libc` does not feature a `uncontext_t` implementation
```

Comment:
Usually we have the comments above `cfg`s

Reply:
aye